### PR TITLE
added rubric_association functions and tests

### LINKED ISSUE #300: Rubric Associations
**What resource needs additional coverage?**
Rubric Associations

**What endpoints need to be covered?**
https://canvas.instructure.com/doc/api/rubrics.html

- [ ] Create a Rubric Association
- [ ] Update a Rubric Association
- [ ] Delete a Rubric Association

### PR DESCRIPTION
**Pull request by weining-li**
# Proposed Changes

Added functions and tests for
* Create a Rubric Association
* Update a Rubric Association
* Delete a Rubric Association

Fixes #300  .

### CODE REVIEW

**Comment by weining-li** [comment]
Apparently I don't know the alphabet 😎

**Comment by Thetwam** [comment]
Hey @weining-li, thanks for the pull request!

It looks like the check for `black` failed. We use `black` to automatically format our code. https://black.readthedocs.io/en/stable/

To install:
```
pip install black
```

To run the formatter:
```
black canvasapi tests
```

Once your code has been blackened, it should be ready for review.

File: canvasapi/course.py [hunk 1]
```diff
@@ -503,6 +523,29 @@ def delete_external_feed(self, feed):
         )
         return ExternalFeed(self._requester, response.json())
 
+    def delete_rubric_association(self, rubric_association, **kwargs):
```
Comment (Thetwam) [inline canvasapi/course.py]: This method should be moved to the RubricAssociation class as `delete()`

File: canvasapi/course.py [hunk 2]
```diff
@@ -2551,6 +2594,30 @@ def update_assignment_overrides(self, assignment_overrides, **kwargs):
             _kwargs=combine_kwargs(**kwargs),
         )
 
+    def update_rubric_association(self, rubric_association, **kwargs):
```
Comment (Thetwam) [inline canvasapi/course.py]: This method should be moved to the RubricAssociation class as `update()`

**Pull request merged**